mbedtls-sys/build/bindgen.rs

```diff
@@ -100,12 +100,14 @@ impl super::BuildConfig {
             };
         }
 
-        
+
         // generate static function wrappers without any other rust related parameters to ensure
         // correctness of result C code
         bindgen::builder()
             .clang_args(cc.get_compiler().args().iter().map(|arg| arg.to_str().unwrap()))
             .header_contents("bindgen-input.h", &header)
+            .allowlist_function("^(?i)mbedtls_.*")
```

Comment:
As discussed, include this in 3.4?

Reply:
Created PR: https://github.com/fortanix/rust-mbedtls/pull/287